app/assets/stylesheets/modules/headline.scss

```diff
@@ -0,0 +1,5 @@
+.headline-error {
+  background: image-url('team-bg.png') center -115px no-repeat #EEE;
```

Comment:
if you're using variables for colors, be consistent and use them on every color ^^